model/src/model.py

```diff
@@ -1,13 +1,42 @@
-import pandas as pd
-from typing import List, Optional, Self
+from typing import Deque, List, Optional, Self
 
 from torch import nn
 from torch.utils.data import DataLoader
 import torch
+import pandas as pd
 import snntorch as snn
+from snntorch import spikegen
 
 from datasets.src.zenke_2a.constants import TEST_DATA_PATH, TRAIN_DATA_PATH
+from datasets.src.zenke_2a.datagen import generate_sequential_dataset
 from datasets.src.zenke_2a.dataset import SequentialDataset
+from model.src.util import MovingAverageLIF, SpikeMovingAverage, TemporalFilter, VarianceMovingAverage
+
+# Zenke's paper uses a theta_rest of -50mV
+THETA_REST = -50
```

Comment:
This likely doesn't make any sense as a value given snnTorch having 0 as lowest mem?